code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -166,6 +166,7 @@
 /obj/item/melee/arm_blade/Initialize(mapload,silent,synthetic)
 	. = ..()
 	ADD_TRAIT(src, TRAIT_NODROP, CHANGELING_TRAIT)
+	ADD_TRAIT(src, TRAIT_DOOR_PRYER, INNATE_TRAIT)
```

Comment:
Can you give the armblade a siemens coefficient of 1 so it can still open shocked doors?

Reply:
It already has a value of 1, the variable is defined as 1 by default under `/obj/item` and isn't changed under `/obj/item/melee`

Reply:
Logging results of discord discussion about this:

>Yes the request was actually to change it to zero to provide insulation for changelings

>However, changelings currently cannot open shocked doors with armblade, so changing that isn't within scope for this PR, even if it does end up being a good change